tracer/src/Datadog.Trace/Configuration/GlobalSettings.cs

```diff
@@ -67,7 +72,7 @@ internal GlobalSettings(IConfigurationSource source)
         /// <param name="enabled">Whether debug is enabled.</param>
         public static void SetDebugEnabled(bool enabled)
```

Comment:
I'm not entirely convinced this switch works when used in an app. May be worth checking (seeing as you're tidying up here!) and marking `[Obsolete]` if not.

Reply:
Actually, I forgot `[Obsolete]` is a breaking change 🙄 so feel free to ignore it

Reply:
Ignoring for now😄